src/clients/migration_proposals.rb

```diff
@@ -1,3 +1,23 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2015 SUSE GmbH, All Rights Reserved.
```

Comment:
Ok, it's just confusing to see several different values there (SUSE Gmbh, SUSE LLC, Novell)...